changelogs/refactor-payment-recommendations-eligibiility

```diff
@@ -0,0 +1,5 @@
+Significance: patch
+Type: Tweak
+Comment: This is a slight refactor on how the recommendations on the payment settings are displayed. I extracted the eligibility part to a separate component, to be reused in an upcoming PR.
```

Comment:
I have not seen this format before with the "Comment:" section.  Usually there is a one line changelog entry beneath here.  Was this manually generated?

Reply:
I used the `npm run changelogger -- add` command specifying that a changelog entry was not needed - this file is what I got as a result :D

Reply:
Ah, that's odd.  I ran it and came up with this:

```
Significance: patch
Type: Tweak

Add payment recommendations eligibility wrapper
```

I'm not sure if the `Comment:` is used in another way as it's a portion I'm unfamiliar with. cc @moon0326

Reply:
It looks like in your case it was entered in the "Changelog entry" step. I left the step blank - let me know if you'd rather want me to add an entry in the changelog

![Screen Shot 2021-08-02 at 3 24 48 PM](https://user-images.githubusercontent.com/273592/127919497-0e3f0872-96aa-4061-8c83-59d57eb40f1c.png)

Reply:
Ya, let's go with a brief entry line instead.  I'm not sure if we have an established routine around handling comments yet.

Reply:
Sounds good! Done in 3bcade35c